docs/using/components/configuration/configuration.rst

```diff
@@ -58,13 +69,18 @@ If one doesn't specify a ``file`` section, the code requires two sections (``vel
 parameter ``no_of_shells``. ``no_of_shells`` is the requested number of shells for a model. The ``velocity`` section
 requires a ``type``. Currently, only ``linear`` is supported.
 
-In the ``densities`` section the ``type`` parameter again decides on the parameters. The type ``uniform`` only needs a
- ``value`` parameter with a density compatible quantity. The type ``branch85_w7`` uses a seven-order polynomial fit to
- the W7 model and is parametrised by time since explosion. The parameters ``time_0`` and ``density_coefficient`` are set
- to sensible defaults and should not be changed.
+.. jsonschema:: schemas/model_definitions.yml#/definitions/density/branch85_w7
 
-The ``abundance`` section again has a possible ``file`` parameter with ``type`` (currently only ``artis`` is allowed)
-and a ``name`` parameter giving a path to a file containing the abundance information.
+.. jsonschema:: schemas/model_definitions.yml#/definitions/density/exponential
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/density/power_law
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/density/uniform
+
+In the ``densities`` section the ``type`` parameter again decides on the parameters. The type ``uniform`` only needs a
+``value`` parameter with a density compatible quantity. The type ``branch85_w7`` uses a seven-order polynomial fit to
+the W7 model and is parametrised by time since explosion. The parameters ``time_0`` and ``density_coefficient`` are set
+to sensible defaults and should not be changed.
```

Comment:
Similarly, make "Densities" subheading under "Specific Structure" (just after schema table) and then put the explanatory lines (L80-83) first and then all density schemas.

Reply:
I do was thinking that after this change we can have a specific writeup for each of the sections created (update existing abstracts & creating where ever required). That will allow for more clarity.